lib/stdlib/test/run_old_pcre1_tests.erl

```diff
@@ -106,8 +106,11 @@ test([{RE0,Line,Options0,Tests}|T],PreCompile,XMode,REAsList) ->
 			_ ->
 			    put(skipped,1)
 		    end,
-		    test(T,PreCompile,XMode,REAsList)
-	    end;
+		    test(T,PreCompile,XMode,REAsList);
+        E ->
+            io:format("~p",[E]),
+            test(T,PreCompile,XMode,REAsList)
+        end;
```

Comment:
This looks like it will hide failed tests.

And it's incorrectly indented. Our tabs are 8 spaces long.

Reply:
removed